recipe/meta.yaml

```diff
@@ -65,7 +66,7 @@ outputs:
 
     # core package & vendored libs use different version scheme than CPP libs, see
     # https://github.com/grpc/grpc/blob/v1.48.1/CMakeLists.txt#L28-L32
-    {% set core_version = "26.0.0" %}
+    {% set core_version = "36.0.0" %}
```

Comment:
yeah, looks like the conda-forge feedstock has a few more tests than here, it would be nice to include them

Reply:
Additionally, have you checked out the [upstream tests](https://github.com/grpc/grpc/blob/v1.59.2/CONTRIBUTING.md#building--running-tests)? it's difficult and time consuming to understand all the details in this feedstock, including patches etc. - I'd be comfortable nevertheless if the upstream tests simply passed